### PR TITLE
feat(timed-backend): add env vars to configure hurricane

### PR DESCRIPTION
**Pull request by fugal-dy**
# Description

This adds the `HURRICANE_REQ_QUEUE_LEN`  environment variable to timed-backend's deployment

# Checklist

* [x] This PR contains a description of the changes I'm making
* [x] I updated the version in Chart.yaml
* [x] I updated applicable README.md files using  `pre-commit run`
* [x] I documented any high-level concepts I'm introducing in `docs/`
* [x] CI is currently green and this is ready for review
* [x] I am ready to test changes after they are applied and released

### CODE REVIEW

File: charts/timed/templates/deployment-backend.yaml [hunk 1]
```diff
@@ -33,6 +33,8 @@ spec:
                   fieldPath: status.podIP
             - name: DJANGO_ALLOWED_HOSTS
               value: '{{ join "," .Values.ingress.hosts | default "localhost" }},$(THIS_POD_IP)'
+            - name: HURRICANE_REQ_QUEUE_LEN
+              value: {{ .Values.backend.hurricane.reqQueueLen }}
```
Comment (hairmare) [inline charts/timed/templates/deployment-backend.yaml:37]: it looks like hurricane values are in settings and not directly in backend leading to the ci fail we're seeing
```suggestion
              value: {{ .Values.backend.settings.hurricane.reqQueueLen }}
```

Comment (fugal-dy) [reply charts/timed/templates/deployment-backend.yaml]: Thanks! should be okay now

Comment (fugal-dy) [reply charts/timed/templates/deployment-backend.yaml]: Ok thanks, that's fixed.

**Comment by hairmare** [comment]
I took this for a spin locally and am seeing the following error in the backend logs:

```
/bin/sh: 1: ./cmd.sh: Permission denied
```
podman confirms that cmd.sh is indeed not executable:

``` 
$ podman run --rm -ti -e bash  ghcr.io/adfinis/timed-backend:v3.0.0-rc2 ls -al cmd.sh
-rw-r--r--. 1 root root 264 Oct 13 16:01 cmd.sh
```

it seems like cmd.sh was chmodded in https://github.com/adfinis/timed-backend/commit/8eb726c6082d5d0b7aa69991114ba2da8f0654c6:

> [cmd.sh](https://github.com/adfinis/timed-backend/commit/8eb726c6082d5d0b7aa69991114ba2da8f0654c6#diff-87560b14da0be98ab081c4c5e14119b10c68b9145364ef5ec0e41047dc1f0ae1) 100755 → 100644

Looks like we'll need a new release that does `chmod 755` on cmd.sh for this to work.

**Review by tongpu — APPROVED** [review]

**Pull request merged**